### PR TITLE
Added exporting cpu/mem limits based on Kubernetes rather than cadvisor

### LINKED ISSUE #399: Export the machine container with the proper CPU limit
Today we convert shares to CPU limit, but this doesn't work for the root container since it always has 1024 shares even though it corresponds to all cores on the machine. We should take the NumCores from the machine and export that instead.

### PR DESCRIPTION
**Pull request by piosz**
Fixes #399

### CODE REVIEW

**Comment by piosz** [comment]
@vmarmol

#399 was only about fixing cpu, but I've also fixed memory to make both resources exported consistently.

File: sources/nodes/types.go [hunk 1]
```diff
@@ -30,6 +30,10 @@ type Info struct {
 	InternalIP string
 	// An optional ID assigned to nodes by either the cloud provider or user.
 	ExternalID string
+	// Cpu capacity of node as number of 1/1024 share of core.
+	CpuCapacity uint64
```
Comment (vmarmol) [inline sources/nodes/types.go]: This is exporting in shares? At the Heapster level (and for nodes) we should probably export in whole cores. Shares is hard to reason about at a node level.

**Comment by vmarmol** [comment]
Outside the issue with shares vs. cores, LGTM

File: sources/nodes/types.go (hunk 1 above)
Comment (piosz) [inline sources/nodes/types.go]: Just wanted to make it consistent with the current unit, but it will be definitely easier to use millicores instead.

**Comment by piosz** [comment]
Changed to use millicores internally. We probably don't want to use whole core (not millicores) to support also machines with fractional part of virtual CPU. Please let me know if it addressed your comment.

File: sinks/api/v1/supported_metrics.go [hunk 2]
```diff
@@ -66,8 +66,7 @@ var statMetrics = []SupportedStatMetric{
 			return spec.HasCpu && (spec.Cpu.Limit > 0)
 		},
 		GetValue: func(spec *cadvisor.ContainerSpec, stat *cadvisor.ContainerStats) []InternalPoint {
-			// Normalize to a conversion factor of 1000.
-			return []InternalPoint{{Value: int64(spec.Cpu.Limit*1000) / 1024}}
+			return []InternalPoint{{Value: int64(spec.Cpu.Limit)}}
```
Comment (vmarmol) [inline sinks/api/v1/supported_metrics.go]: I would have expected we still do this? We fixed it for node containers but not for general containers no?

Comment (piosz) [inline sinks/api/v1/supported_metrics.go]: I see the problem. Fixed.

**Comment by piosz** [comment]
ping

**Comment by vmarmol** [comment]
LGTM, sorry this dropped out of my radar!

**Comment by piosz** [comment]
No worries. Thanks you!

**Pull request merged**